cmd/jag/main.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 var (
-	date       = "2022-01-20T04:28:18Z"
-	version    = "v0.7.3"
-	sdkVersion = "v0.15.1"
+	date       = "2022-01-31T07:01:28Z"
+	version    = "8a66bb2"
```

Comment:
Is this the version of Jaguar?
If yes, using one that is increasing would be better.

Reply:
This file is maintained by a script up until we're ready to release (which is when we add the version that we will use as the tag).